datadog_checks_dev/datadog_checks/dev/plugin/tox.py

```diff
@@ -15,7 +15,7 @@
 MYPY_ARGS_OPTION = 'dd_mypy_args'
 E2E_READY_CONDITION = 'e2e ready if'
 FIX_DEFAULT_ENVDIR_FLAG = 'ensure_default_envdir'
-ISORT_DEP = 'isort[pyproject]==4.3.21'  # cap isort due to https://github.com/timothycrosley/isort/issues/1278
+ISORT_DEP = 'isort[pyproject]==5.*'
```

Comment:
It's probably better to pin to a specific version. Otherwise, it might break CI and will need **immediate/urgent** attention, which is something we want to avoid.

Bumping manually is likely preferable for style/lint.

Reply:
I personally prefer breakages so we are always up-to-date on things